1/assignment-week1.py

```diff
@@ -0,0 +1,83 @@
+print ("Week 1 Assignment - Math Equation Evaluator")
+print ("Intern - Shu Wei")
+print ("")
+
+eArray = list(map(str, input("Please enter your input with spaces between them: ").split()))
+
+#Definitions
+aCount = (len(eArray))
+fCount = 3
+nCount = 4
+needle = 0
+
+# Check whether the first two inputs are numbers and the last input is a sign
+try:
+    check1 = int(eArray[0])
+except ValueError:
+    print("Please make sure that the input #1 is a number")
+    exit()
+try:
+    check2 = int(eArray[1])
+except ValueError:
+    print("Please make sure that the input #2 input is a number")
+    exit()
+
+if aCount > 3:
+    CheckNCount = 4
+    while (CheckNCount <= aCount):
+        try:
+            checkN = int(eArray[CheckNCount-1])

```

Comment:
since you were doing validation earlier on `eArray` you either converted its values to `int` already or you'd already failed it. so  there's no point running `int` here